src/core/accido/_class_noun.py

```
@@ -5,9 +5,10 @@
 import logging
 from functools import total_ordering
 from typing import TYPE_CHECKING, Final, overload
+from warnings import deprecated
 
 from ._class_word import _Word
-from .edge_cases import IRREGULAR_NOUNS
+from .edge_cases import IRREGULAR_DECLINED_NOUNS, IRREGULAR_NOUNS
```

Pylint (3.3.7)

[error] Cannot import 'edge_cases' due to 'invalid syntax (src.core.accido.edge_cases, line 228)'

(E0001)